src/icalendar/prop.py

```diff
@@ -1100,6 +1100,68 @@ def __reduce_ex__(self, _p):
 
 class vRecur(CaselessDict):
     """Recurrence definition.
+
+    Property Name: RRULE
```

Comment:
All parameter names must appear on their own line, followed by an optional description beginning on an indented new line. Please adjust the others accordingly. ``Except`` is OK. See current rendering at https://icalendar--759.org.readthedocs.build/en/759/api.html#icalendar.prop.vRecur

```suggestion
    Property Name:
        RRULE
```

Reply:
The rendering now looks good